packages/vhd-lib/src/openVhd.js

```diff
@@ -0,0 +1,15 @@
+import { VhdFile, VhdDirectory } from './'
+
+export const openVhd = async (handler, path) => {
```

Comment:
I see no reasons to use arrow function here.

```suggestion
export async function openVhd (handler, path) {
```